pkg/reconciler/v1alpha1/eventlistener/eventlistener.go

```diff
@@ -543,7 +538,7 @@ func getContainer(el *v1alpha1.EventListener, c Config) corev1.Container {
 		Args: []string{
 			"--el-name=" + el.Name,
 			"--el-namespace=" + el.Namespace,
-			"--port=" + strconv.Itoa(port),
+			"--port=" + strconv.Itoa(eventListenerContainerPort),
```

Comment:
The idea was together with @dibyom :
Changing the port for the deployment / container to be static while only making the `service` configurable. This causes two effects.
1. Less work templating because we only need to change the port in the `service` instead of everywhere.
2. Logic becomes easier to understand because configuration is only applied to the `service` and not to the deployment / container and so on.

I chose port `8000` because noone from outside should consume it - they should interact with the service instead (where it remains `8080` / `8443` by default). Additionally it makes clear (to me at least) that it could be either protocol (8080 implies HTTP).